src/OrbitGl/CaptureViewElementTest.cpp

```diff
@@ -67,4 +100,85 @@ TEST(CaptureViewElementTesterTest, PassesAllTestsOnExistingElement) {
                                                      tester.GetLayout(), kChildCount);
   tester.RunTests(&container_elem);
 }
+
+const Viewport kViewport(100, 100);
+const TimeGraphLayout kLayout;
+
+TEST(CaptureViewElement, IsMouseOverWorksForVisibleElements) {
+  UnitTestCaptureViewLeafElement elem(nullptr, &kViewport, &kLayout);
+  elem.SetWidth(kViewport.GetWorldWidth());
+
+  EXPECT_FALSE(elem.IsMouseOver(Vec2(-1, -1)));
+  EXPECT_TRUE(elem.IsMouseOver(Vec2(5, 5)));
+  EXPECT_TRUE(elem.IsMouseOver(elem.GetSize()));
+  EXPECT_TRUE(elem.IsMouseOver(Vec2(0, 0)));
+}
+
+TEST(CaptureViewElement, IsMouseOverDoesNotWorkForElementsOutsideOfTheParent) {
```

Comment:
I imagine that this is needed for things outside of their parent, but I would rather return true in IsMouseOver and never send the event to its children. Therefore, I would erase this test.